extensions/reviewed/BoidsMovement.json

```diff
@@ -743,7 +753,7 @@
                 "const object = objects[0];",
                 "const behaviorName = eventsFunctionContext.getBehaviorName(\"Behavior\");",
                 "const behavior = object.getBehavior(behaviorName);",
-                "runtimeScene.__boidsExtension.boidsManager.remove(behavior);"
+                "behavior.activated = 0;"
```

Comment:
By removing this line, deactivated Boids will stays in the R-Tree and will continue to have side effect on other Boids.
A behavior already have a `activated()` accessor, but it's not needed. What should probably be done is to remove the Boids from `boids`.

Reply:
that's the error, by removing the boid from the R-tree it forgets its original behavior when reactivated, so my solution is to pause and then resume it when needed